src/execution/utils.h

```diff
@@ -11,7 +11,13 @@ bool VerifyKeyInThisShard(RedisModuleCtx *ctx, RedisModuleString *key_str);
 /**
  * Use this function when loading the model. Stores the version in global variables.
  */
-void getRedisVersion();
+void setRedisVersion();
```

Comment:
```suggestion
void redisai_getRedisVersion();
```
if you change the name, let the reader know it is a module-level function